R/deprecated.R

```diff
@@ -20,14 +20,12 @@ NULL
 #'
 #' @return A list with
 #' \itemize{
-#' \item results as a data.frame (`dplyr` `tbl_df`) called results with one line
-#'  per results,
+#' \item results as a tibble/data.frame with one line per result,
```

Comment:
```suggestion
#' \item results as a tibble with one line per result,
```

tibbles are not data.frames according to the longest [R-pkg-devel thread ever](https://stat.ethz.ch/pipermail/r-package-devel/2017q3/001896.html). Hence my not being sure we could use "tibble/data.frame" without making someone angry.

Reply:
I wasn't aware of this thread. It is long indeed and ... interesting. Not sure, whether I agree that "tibbles are not data.frames" based on the evidence presented there, but that does not matter and it does no harm to omit "data.frame" here.